CHANGELOG.rst

```diff
@@ -4,7 +4,7 @@ Changelog
 0.1.5 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Fix: Limit decimal values to 6 places when sending a value to a charge point
```

Comment:
I think we should keep OCPP's naming here and refer to a "charging station" rather than to a "charge point".